sandbox/modules/custom-codecs/src/main/java/org/opensearch/index/codec/customcodec/Lucene92CustomCodec.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index.codec.customcodec;
+
+import org.apache.lucene.codecs.StoredFieldsFormat;
+import org.apache.lucene.codecs.FilterCodec;
+import org.apache.lucene.codecs.lucene92.Lucene92Codec;
```

Comment:
@mulugetam I think we need to update to `Lucene94Codec` here and in other places, `main` is on Apache Lucene 9.5 now ....

Reply:
@reta why not `Lucene95Codec`?

Reply:
@mulugetam even better

Reply:
@reta now using `Lucene95Codec`.